src/patternfly/components/DataList/data-list.scss

```diff
@@ -238,6 +247,68 @@
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
   }
 
+  // stylelint-disable
+  &.pf-m-draggable {
+    position: relative;
+    cursor: grab;
+
```

Comment:
```suggestion

    &:active {
      cursor: grabbing;
    }
```

Reply:
i added this below, should i move it above hover ?

Reply:
Yeah, that's probably better